tests/test_graphql_farmers.py

```
@@ -0,0 +1,136 @@
+        id
+        farmName
+        location { latitude longitude city }
+      }
+    }
+    """
+
+    f1 = await _graphql(client, create_mut, {
+        "input": {
+          "userId": "00000000-0000-0000-0000-000000000001",
+          "farmName": "Andes Farm",
+          "organicCertified": true,
```

Ruff (0.12.2)

Undefined name `true`

(F821)